.releaserc

```diff
@@ -0,0 +1,77 @@
+{
+  "branches": ["master"],
+  "repositoryUrl": "https://github.com/genesiscommunitysuccess/custom-elements-lsp",
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "preset": "conventionalcommits",
+        "releaseRules": [
+          {
+            "type": "*!",
+            "release": "major"
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "conventionalcommits",
+        "presetConfig": {
+					"issuePrefixes": ["FUI-", "ISSUE-"],
+					"issueUrlFormat": "",
+					"linkReferences": false,
+        },
+        "releaseRules": [
+          {
+            "type": "*!",
+            "release": "major"
+          }
+        ]
+      }
+    ],
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "CHANGELOG.md",
+        "changelogTitle": "# Changelog"
+      }
+    ],
+    [
+      "@google/semantic-release-replace-plugin",
```

Comment:
In foundation-ui we need this because of Lerna quirks, however here `npm version` will update this file anyway.
You can also remove this dependency then.